src/pages/Product/components/Review/Review.js

```diff
@@ -13,17 +14,28 @@ const Review = () => {
       </div>
       <div className="reviewTitle">배송도 빠르고 정말 좋아요.</div>
       <div className="reviewMain">
-        <div className="reviewMainContents">
-          주문한지 하루만에 배송이 도착해서 너무 좋았습니다.
-          <br />
-          다음에도 필요하면 또 여기서 시킬 것 같아요.
-        </div>
-        <span className="reviewMainHandler" onClick={() => {}}>
-          더보기 ▼
+        {review === true ? <ReviewMainContents /> : null}
+        <span
+          className="reviewMainHandler"
+          onClick={() => {
+            setReview(!review);
+          }}
+        >
+          {review === true ? "접기 ▲" : "더보기 ▼"}
```

Comment:
```suggestion
          {review ? "접기 ▲" : "더보기 ▼"}
```

review 값 자체가 boolean 값이기 때문에 조건식이 불필요합니다!